README.md

```diff
@@ -30,19 +31,33 @@ query and filter sample data via REST.
 
 #### Application information
 
-- Name: `loopback-example-model-relations`
-- Directory to contain the project: `loopback-example-model-relations`
-
 ```
 $ slc loopback loopback-example-model-relations
 ... # follow the prompts
+- Name: `loopback-example-model-relations`
+- Directory to contain the project: `loopback-example-model-relations`
 $ cd loopback-example-model-relations
 ```
+### Create the datasource
+
+```
+$ slc loopback:datasource
+... # follow the prompts, choose `other` to define custom connctor
+```
+- Name: `transient`
+- Connector: `other`
+  - Name: `transient`
 
 ### Create the models
 
 #### Model information
 
+```
+$ slc loopback:model Customer
+... # follow the prompts, repeat for other models
```

Comment:
Here also. We should be consistend in listing the data for the prompt first, or the data for the prompt after in all steps.